test/resourceNameTest.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
```

Comment:
You need three more route tests as this test only has the GET/POST for 200 status codes. Here's a reminder of the test requirements:

```
GET - test 404, responds with 'not found' for valid request made with an id that was not found
GET - test 400, responds with 'bad request' if no id was provided in the request
GET - test 200, response body like {<data>} for a request made with a valid id
POST - test 400, responds with 'bad request' for if no body provided or invalid body
POST - test 200, response body like {<data>} for a post request with a valid body
```